app/controllers/friendships_controller.rb

```diff
@@ -0,0 +1,20 @@
+class FriendshipsController < ApplicationController
+  def create
+    @friendship = current_user.friendships.build(friend_id: params[:friend_id])
```

Comment:
If signed out, trying to add a friend returns a NoMethodError on line 3. Would probably need to return a flash notice saying you need to sign up/ log in to add friends.